src/controllers/livetvstatus.js

```diff
@@ -1,294 +1,307 @@
-define(['jQuery', 'globalize', 'scripts/taskbutton', 'dom', 'libraryMenu', 'layoutManager', 'loading', 'browser', 'listViewStyle', 'flexStyles', 'emby-itemscontainer', 'cardStyle', 'material-icons', 'emby-button'], function ($, globalize, taskButton, dom, libraryMenu, layoutManager, loading, browser) {
-    'use strict';
-
-    var enableFocusTransform = !browser.slow && !browser.edge;
-
-    function getDeviceHtml(device) {
-        var padderClass;
-        var html = '';
-        var cssClass = 'card scalableCard';
-        var cardBoxCssClass = 'cardBox visualCardBox';
-        cssClass += ' backdropCard backdropCard-scalable';
-        padderClass = 'cardPadder-backdrop';
-
-        // TODO move card creation code to Card component
-
-        if (layoutManager.tv) {
-            cssClass += ' show-focus';
-
-            if (enableFocusTransform) {
-                cssClass += ' show-animation';
-            }
+'use strict';
+
```

Comment:
We want to remove this as well.

Reply:
'use strict` why?

Reply:
ES6 modules are automatically in `strict` mode see [here](https://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code)

Reply:
Fixed